packages/stateful/actions/core/dao_governance/AcceptSubDao/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { FamilyEmoji } from '@dao-dao/stateless'
+import {
+  ActionComponent,
+  ActionKey,
+  ActionMaker,
+  UseDecodedCosmosMsg,
+  UseDefaults,
+} from '@dao-dao/types/actions'
+import { makeWasmMessage } from '@dao-dao/utils'
+
+import { AddressInput } from '../../../../components'
+import { AcceptSubDaoComponent, AcceptSubDaoData } from './Component'
+
+const defaultAcceptSubDaoData = {
+  address: '',
+}
+
+const useDefaults: UseDefaults<AcceptSubDaoData> = () => defaultAcceptSubDaoData
+
+const useDecodedCosmosMsg: UseDecodedCosmosMsg<AcceptSubDaoData> = (
+  msg: Record<string, any>
+) => {
+  try {
+    return {
+      match: true,
+      data: {
+        address: msg.wasm.execute.contract_addr,
+      },
+    }
+  } catch (e) {
```

Comment:
```suggestion
  } catch {
```

drop the unused variable